process/templates/PROJECT_NAME_sandbox_stage.md

```diff
@@ -4,6 +4,10 @@
 The project must have a minimum of two maintainers with different organizational affiliations.
   * "name, affiliation, GitHub ID"
 
+### Sponsor
+Most projects will be sponsored by an existing OpenSSF Working Group, although in some cases a project may be sponsored by the TAC. The project commits to providing the sponsor quarterly updates on progress.
+  * "name of the project sponsor: either a Working Group or the TAC"
+
```

Comment:
I think this is using the term "sponsor" in a way that is different from what we've used it so far. Indeed, we've used the term to refer to a *person*. For instance, the project lifecycle states that a project "Receives a TAC and WG sponsor for guidance on technical direction."
This text uses "sponsor" to refer to the entity the project reports to.

Reply:
This is exactly the question - I think we're trying to describe the reporting relationship. For SIGs we use the term "governing body" (which feels maybe too formal, but at least is clear). For projects I though we were using the term "sponsor" in the same way we use "governing body" for SIGs.

Maybe @SecurityCRob can clarify? Are we referring to the reporting relationship here? Or is a sponsor something else that is separate from the reporting relationship?

Reply:
I never used it that way but maybe it's a matter of interpretation and we all have different perspectives on that one...
I too feel that "governing body" is a bit strong, that's why I phrased the above as "the entity the project reports to" instead. :-) It seems a bit softer to me.

Reply:
How about "deliverable" as a better word for this - as in "projects are deliverables of working groups..."

Reply:
> This is exactly the question - I think we're trying to describe the reporting relationship. For SIGs we use the term "governing body" (which feels maybe too formal, but at least is clear). For projects I though we were using the term "sponsor" in the same way we use "governing body" for SIGs.
> 
> Maybe @SecurityCRob can clarify? Are we referring to the reporting relationship here? Or is a sponsor something else that is separate from the reporting relationship?

Reporting relationship